cli/src/commands/branches/delete.ts

```diff
@@ -24,14 +24,15 @@ export default class BranchesDelete extends BaseCommand {
 
     const { confirm } = await this.prompt(
```

Comment:
Non-blocking idea:

This pattern appears 3x in this PR, we could factor out a function called `confirmWithPhrase` or something and reuse it.

```
const { confirm } = await this.prompt(
      {
        type: 'text',
        name: 'confirm',
        message: `${message} Please type ${confirmPhrase} to confirm`,
        initial: false
      },
      flags.force ? {confirmPhrase} : undefined
    );
```